lib/noble_page.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require_relative 'member_page'
+
+class NoblePage < MemberPage
+  field :constituency do
+    table_field('Constituency').match(' for (.*)').to_a[1]
+  end
+
+  field :cell do
+    noko.xpath("//ul/li[contains(., 'Mob number')]").text.split(':').last.to_s.tidy
```

Comment:
I think this needs to comment to explain (a) why it needs to be overridden from the parent, and (b) what the layout on the page. (Trying to derive what the page probably looks like from this code is an interesting exercise, and is significantly more difficult than if the `contains` had been 'Mob number:' (with a colon) and the split had been a `split(':', 2)`, for example)